configure.ac

```diff
@@ -114,7 +113,7 @@ eval libdir="`eval echo ${libdir}`"
 eval infodir="`eval echo ${infodir}`"
 eval mandir="`eval echo ${mandir}`"
 
-AM_CONDITIONAL([PREFIX_IS_USR], [test "${prefix}" = "/usr"])
+AM_CONDITIONAL([UNAME_IS_DEBIAN], [test x`uname -v | grep -oh Debian` = x"Debian"])
```

Comment:
This is a check which reads the 

> version of the LSB specification against which the distribution is compliant.

It works on RHEL and Debian - and should work on any Linux distribution.  It enables the `if`/`fi` in `Makefile.am`